src/components/navigation/Header.svelte

```diff
@@ -1,9 +1,15 @@
 <script>
     import { PUBLIC_OBA_REGION_NAME } from '$env/static/public';
+    import ThemeSwitcher from '$lib/ThemeSwitch/ThemeSwitcher.svelte';
+
+    import '../../app.css';
```

Comment:
isn't this already included in the application layout?

Reply:
Yeah, removed from the Header.svelte